core/modules/utils/dom/http.js

```diff
@@ -34,6 +34,23 @@ exports.httpRequest = function(options) {
 			});
 			return result;
 		},
+		getHeader = function(targetHeader) {
+			return headers[targetHeader] || headers[targetHeader.toLowerCase()];
+		},
+		isSimpleRequest = function(type,headers) {
+			if(["GET","HEAD","POST"].indexOf(type) === -1) {
```

Comment:
Should there be a `indexOf(type.toUpperCase())` ... just in case?

Reply:
HTTP methods (verbs) are case sensitive, whereas headers are not.